src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
CCompiler Spawn is a documented part of the distuitils API, so its use shouldn't change in the Py3 line https://docs.python.org/3/distutils/apiref.html

I would argue that the bug is in ```spawn()``` to begin with, the issue in ```link()``` is just a symptom because it happens to generate a large enough command for spawn.  I think we should just pipe all our commands to a file, and then use the file, and then use the command file hack for everything.